src/Numerics/DGMethods/DGModel_kernels.jl

```diff
@@ -25,17 +25,46 @@ const _sM, _vMI = Grids._sM, Grids._vMI
 # }}}
 
 @doc """
-    volume_tendency!(balance_law::BalanceLaw, Val(polyorder),
-                     tendency, state_prognostic, state_gradient_flux,
-                     state_auxiliary, vgeo, t, D, elems)
-
-Computational kernel: Evaluate the volume integrals on right-hand side of a
-`BalanceLaw` semi-discretization.
+    function volume_tendency!(
+        balance_law::BalanceLaw,
+        ::Val{dim},
+        ::Val{polyorder},
+        model_direction,
+        direction,
+        tendency,
+        state_prognostic,
+        state_gradient_flux,
+        Qhypervisc_grad,
+        state_auxiliary,
+        vgeo,
+        t,
+        ω,
+        D,
+        elems,
+        α,
+        β,
+        add_source = false,
+    )
+
+Compute kernel for evaluating the volume tendencies for the
+DG form:
+
+∫ₑ ψ⋅ ∂q/∂t dx - ∫ₑ ∇ψ⋅(Fⁱⁿᵛ + Fᵛⁱˢᶜ) dx + ∮ₑ n̂ ψ⋅(Fⁱⁿᵛ* + Fᵛⁱˢᶜ*) dS,
+
+or equivalently in matrix form:
+
+dQ/dt = M⁻¹(MS + DᵀM(Fⁱⁿᵛ + Fᵛⁱˢᶜ) + ∑ᶠ Mfᵀ(Fⁱⁿᵛ* + Fᵛⁱˢᶜ*)).
```

Comment:
```suggestion
dQ/dt = M⁻¹(MS + DᵀM(Fⁱⁿᵛ + Fᵛⁱˢᶜ) + ∑ᶠ Lᵀ * Mf * (Fⁱⁿᵛ* + Fᵛⁱˢᶜ*)).
```
Why the transpose on `Mfᵀ`? Also, I think there should be an interpolation here from the volume to the face (see https://github.com/CliMA/ClimateMachine.jl/commit/ad078be755491c60362d24bdc9c02433aa36083a#r43809685)

(Same comment applies to all equations below)

Reply:
Whoops that's an error on my part --- will correct them!